docs/literate/src/files/index.jl

```diff
@@ -56,6 +56,18 @@
 # explained and added to an exemplary simulation of the Sedov blast wave with the 2D compressible Euler
 # equations.
 
+#src Note to developers: Use "{ index }" (but without spaces, see next line) to enable automatic indexing
+# ### [{index} Subcell limiting with the IDP Limiter](@ref subcell_shock_capturing)
+#-
+# Trixi.jl features a subcell-wise limiting strategy utilizing an Invariant Domain-Preserving (IDP)
```

Comment:
Maybe provide already a link to a paper that introduced IDP here?

Reply:
We don't ever link papers on this page (only other parts of the docs and software). I think it's better to keep it this way. But I don't have a strong opinion on this.